types/xovis/PC2SV5/gender.schema.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$id": "https://akenza.io/xovis/pc2sv5/gender.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Gender",
+  "type": "object",
+  "properties": {
+    "gender": {
```

Comment:
how is this data intended to be used? 
what if multiple people are in the view point?

Reply:
The sensor tracks each person as a separate entity and only assigns gender on a line crossing. Worst case its unknown.